{
   "active_lock_reason" : null,
   "assignee" : null,
   "assignees" : [],
   "author_association" : "NONE",
   "body" : "### Is there an existing issue for this?\n\n- [X] I have searched the existing issues\n\n### Current behaviour\n\nHello, I installed a new wallet for myself, the blocks started updating to the latest version, but suddenly the computer rebooted and now I canât open my wallet. The error appears: \"berkeley database failed to open database environment\"\r\n![2024-01-20_11-19-18](https://github.com/bitcoin/bitcoin/assets/38570093/9db96736-2098-436f-ae49-62b19d9115b0)\r\n\n\n### Expected behaviour\n\nHello, I installed a new wallet for myself, the blocks started updating to the latest version, but suddenly the computer rebooted and now I canât open my wallet. The error appears: \"berkeley database failed to open database environment\"\r\n![2024-01-20_11-19-18](https://github.com/bitcoin/bitcoin/assets/38570093/9db96736-2098-436f-ae49-62b19d9115b0)\n\n### Steps to reproduce\n\nHello, I installed a new wallet for myself, the blocks started updating to the latest version, but suddenly the computer rebooted and now I canât open my wallet. The error appears: \"berkeley database failed to open database environment\"\r\n![2024-01-20_11-19-18](https://github.com/bitcoin/bitcoin/assets/38570093/9db96736-2098-436f-ae49-62b19d9115b0)\n\n### Relevant log output\n\n_No response_\n\n### How did you obtain Bitcoin Core\n\nCompiled from source\n\n### What version of Bitcoin Core are you using?\n\nv26\n\n### Operating system and version\n\nWindows 11pro\n\n### Machine specifications\n\n_No response_",
   "closed_at" : null,
   "closed_by" : null,
   "comments" : 1,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29286/comments",
   "created_at" : "2024-01-20T08:23:30Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29286/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/29286",
   "id" : 2091962484,
   "labels" : [],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29286/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "node_id" : "I_kwDOABII5858sNB0",
   "number" : 29286,
   "performed_via_github_app" : null,
   "reactions" : {
      "+1" : 0,
      "-1" : 0,
      "confused" : 0,
      "eyes" : 0,
      "heart" : 0,
      "hooray" : 0,
      "laugh" : 0,
      "rocket" : 0,
      "total_count" : 0,
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29286/reactions"
   },
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "open",
   "state_reason" : null,
   "timeline_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29286/timeline",
   "title" : "berkeley database failed to open database environment",
   "updated_at" : "2024-01-20T08:27:15Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/29286",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/38570093?v=4",
      "events_url" : "https://api.github.com/users/freezoloto/events{/privacy}",
      "followers_url" : "https://api.github.com/users/freezoloto/followers",
      "following_url" : "https://api.github.com/users/freezoloto/following{/other_user}",
      "gists_url" : "https://api.github.com/users/freezoloto/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/freezoloto",
      "id" : 38570093,
      "login" : "freezoloto",
      "node_id" : "MDQ6VXNlcjM4NTcwMDkz",
      "organizations_url" : "https://api.github.com/users/freezoloto/orgs",
      "received_events_url" : "https://api.github.com/users/freezoloto/received_events",
      "repos_url" : "https://api.github.com/users/freezoloto/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/freezoloto/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/freezoloto/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/freezoloto"
   }
}
